spec/rubocop/cop/style/empty_case_condition_spec.rb

```diff
@@ -167,5 +167,29 @@
 
       it_behaves_like 'detect/correct empty case, accept non-empty case'
     end
+
+    context 'with first when branch including comma-delimited alternatives' do
```

Comment:
@bbatsov This particular issue happened because autocorrect of `when` branches was done for all branches except the first.
`when_nodes.shift` removed the first one from the array before the correction function had a chance to fix it, I replaced it with `when_nodes.first`.
One more thing which needed to be fixed was to replace `elsif` starting from second `when` branch instead of first - which was the case before due to the `shift`, thus the `when_nodes[1..-1]`).
If you look at the spec, there were already passing tests with comma-delimited alternatives, but not in the first `when` branch.